src/ripple/basics/Buffer.h

```diff
@@ -40,29 +40,8 @@ class Buffer
 
 public:
     Buffer() = default;
-
-    Buffer& operator= (Buffer const& other)
-    {
-        size_ = other.size_;
-        if (size_)
-        {
-            p_.reset (new std::uint8_t[size_]);
-            memcpy (p_.get(), other.p_.get(), size_);
-        }
-        else
-            p_.reset();
-        return *this;
-    }
-
-    Buffer (Buffer const& other)
-        : size_ (other.size_)
-    {
-        if (size_)
-        {
-            p_.reset (new std::uint8_t[size_]);
-            memcpy (p_.get(), other.p_.get(), size_);
-        }
-    }
```

Comment:
@vinniefalco 
* https://github.com/ripple/rippled/pull/1840/commits/d3c00d019fdb0fd49abaf9a2378d6e7c9c8135ad#r78318778
* https://github.com/ripple/rippled/pull/1840/commits/d3c00d019fdb0fd49abaf9a2378d6e7c9c8135ad#r78318881

I don't know that the problem was unrecoverable, but here we are.